src/rgw/rgw_perf_counters.h

```diff
@@ -4,25 +4,26 @@
 #pragma once
 
 #include "include/common_fwd.h"
+#include "rgw_common.h"
+#include "rgw_sal.h"
+#include "common/perf_counters_cache.h"
+#include "common/perf_counters_key.h"
 
 extern PerfCounters *perfcounter;
+extern ceph::perf_counters::PerfCountersCache *user_counters_cache;
+extern ceph::perf_counters::PerfCountersCache *bucket_counters_cache;
+extern std::string rgw_op_counters_key;
 
 extern int rgw_perf_start(CephContext *cct);
 extern void rgw_perf_stop(CephContext *cct);
+extern void frontend_counters_init(CephContext *cct);
+extern std::shared_ptr<PerfCounters> create_rgw_counters(const std::string& name, CephContext *cct);
```

Comment:
i don't think these need to be exposed in the header, if they're only called by `rgw_perf_start()` in rgw_perf_counters.cc